core/src/main/java/hudson/logging/LogRecorder.java

```diff
@@ -421,8 +421,9 @@ private XmlFile getConfigFile() {
      * @since 1.519
      */
     public Map<Computer,List<LogRecord>> getSlaveLogRecords() {
-        Map<Computer,List<LogRecord>> result = new TreeMap<Computer,List<LogRecord>>(new Comparator<Computer>() {
+        Map<Computer,List<LogRecord>> result = new TreeMap<>(new Comparator<Computer>() {
             final Collator COLL = Collator.getInstance();
+
```

Comment:
In case anyone else is curious about the lines changed, this is the one causing the imbalance.